### DISCUSSION TITLE
Where to sign in TrueNas

### DISCUSSION THREAD

**Discussion by DROPLOTHUB** [AdguardTeam/AdGuardHome#4031] [Q&A]
what are the credentials to sign in on as i cant create a username and password

**Comment by EugeneOne1** [comment]
@DROPLOTHUB, hello.  What exactly do you mean by this:

 >  i cant create a username and password

You don't need any credentials unless the initial setup where these credentials are set is performed.

**Comment by DROPLOTHUB** [comment]
well installed it on my truenas as iocage went to the admin portal and its asking for a username and password

**Comment by EugeneOne1** [comment]
@DROPLOTHUB, well, I'm not really familiar with the described setup.  Have you got an access to the container's contents, e.g. via ssh?  If yes, it's possible to change credentials through the `AdGuardHome.yaml` configuration file as described [here](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#password-reset).
